### PR TITLE
[aws][feat] Add additional policies to the cloudwatch 

### PR DESCRIPTION
**Pull request by 1101-1**
# Description

# To-Dos

- [x] I have created tests for any new or updated functionality.
- [x] I ran `tox` successfully.

# Code of Conduct

By submitting this pull request, I agree to follow the [code of conduct](https://inventory.fix.security/code-of-conduct).

### CODE REVIEW

File: plugins/aws/fix_plugin_aws/resource/bedrock.py [hunk 1]
```diff
@@ -549,7 +551,7 @@ class AwsBedrockModelCustomizationJob(BedrockTaggable, BaseAIJob, AwsResource):
         "bedrock",
         "list-model-customization-jobs",
         "modelCustomizationJobSummaries",
-        expected_errors=["ValidationException"],
+        expected_errors=["ValidationException", "InternalServerException"],
```
Comment (aquamatthias) [inline plugins/aws/fix_plugin_aws/resource/bedrock.py:554]: InternalServerException: sounds like something that should be retried and not ignored?

File: plugins/aws/fix_plugin_aws/resource/bedrock.py [hunk 2]
```diff
@@ -776,7 +778,10 @@ class AwsBedrockEvaluationJob(BedrockTaggable, BaseAIJob, AwsResource):
         "successors": {"default": [AwsS3Bucket.kind, AwsKmsKey.kind]},
     }
     api_spec: ClassVar[AwsApiSpec] = AwsApiSpec(
-        "bedrock", "list-evaluation-jobs", "jobSummaries", expected_errors=["AccessDeniedException"]
+        "bedrock",
+        "list-evaluation-jobs",
+        "jobSummaries",
+        expected_errors=["AccessDeniedException", "InternalServerException"],
```
Comment (aquamatthias) [inline plugins/aws/fix_plugin_aws/resource/bedrock.py:784]: same here

File: plugins/aws/fix_plugin_aws/resource/cloudwatch.py [hunk 3]
```diff
@@ -364,11 +365,75 @@ class AwsCloudwatchLogGroup(LogsTaggable, AwsResource):
...
+                        policy_statements = [policy_statements]
+
+                    for statement in policy_statements:
+                        statement_resources = statement.get("Resource")
+                        log_group_arns = parse_resource_arn(statement_resources)
+
+                        if any(is_arn_match(arn, target_group_arn) for arn in log_group_arns):
+                            # If a match is found, associate the policy and move to the next policy
+                            associated_policies[policy_name] = policy
+                            break
```
Comment (aquamatthias) [inline plugins/aws/fix_plugin_aws/resource/cloudwatch.py:408]: @meln1k Is there something in your access edges PR that would simplify this logic?

Comment (meln1k) [reply plugins/aws/fix_plugin_aws/resource/cloudwatch.py:408]: unfortunately not.

**Review by aquamatthias — APPROVED** [review]

**Pull request merged**